packages/editor/src/components/editor/index.js

```diff
@@ -115,6 +142,7 @@ function Editor( {
 					settings={ settings }
 					initialEdits={ initialEdits }
 					useSubRegistry={ false }
+					initialSelection={ initialSelection }
```

Comment:
Is this prop doing anything? I thought we would pass down the initial selection as a fallback or in place of the `getEditorSelection` value in the Editor component.

https://github.com/WordPress/gutenberg/blob/b9e9788296b1973ae57c98edcf68c14e4f113104/packages/editor/src/components/provider/index.js#L368